src/stack_pr/cli.py

```diff
@@ -948,6 +948,10 @@ def land_pr(e: StackEntry, remote: str, target: str):
         input=pr_body.encode(),
     )
 
+def abandon_pr(e: StackEntry):
+    log(b("Abandoning ") + e.pprint(), level=2)
+
+    run_shell_command(["gh", "pr", "close", e.pr])
```

Comment:
`e.pr` can be empty (e.g. if a new commit has been added to the stack), so we should handle it here to prevent a crash.